app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -52,7 +52,7 @@ def handle_valid_otp
   end
 
   def authentication_context?
-    context == 'authentication'
+    context == 'authentication' || context == 'reauthentication'
```

Comment:
maybe now is still not the time...but I think it would be great if we could make these auth contexts into constants? Making them constants helps remind that they're finite and enumerable